ebs/ebs.go

```diff
@@ -589,6 +598,15 @@ func (d *Driver) CreateVolume(req Request) error {
 		if err := fs.FormatDevice(volume.Device, d.DefaultFSType); err != nil {
 			return err
 		}
+		log.Debugf("Launching snapshot creation for brand new volume: %s", volumeName)
+		request := &CreateSnapshotRequest{
```

Comment:
I'm open to that. Depends on how much we want to abstract away. Was trying to change as little convoy code as possible, though at this point, maybe that is a lost cause :)